CHANGELOG.md

```diff
@@ -5,6 +5,148 @@
 ## New features
 - feat: Added new tracing API (**HIGHLY EXPERIMENTAL**) supporting two RPC methods: `trace_block` and `trace_replayBlockTransactions` ([filecoin-project/lotus#11100](https://github.com/filecoin-project/lotus/pull/11100))
```

Comment:
I think this is captured in your. release! see line 15 - so remove from the unreleased section?